packaging/gunicorn/ovirt_logging_path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/logging.yaml b/logging.yaml
+index 8357727..a883306 100644
+--- a/logging.yaml
++++ b/logging.yaml
+@@ -17,7 +17,7 @@ handlers:
+         class: logging.handlers.RotatingFileHandler
+         level: DEBUG
+         formatter: simple
+-        filename: ansible-runner-service.log
++        filename: /var/log/ovirt-engine/ansible-runner-service.log
```

Comment:
Shouldn't we create /var/log/ovirt-engine directory as a part of -dev RPM installation?

Reply:
That is older commit have migrated the patch to https://github.com/ansible/ansible-runner-service/pull/70